2022-07-15T16:00:35.801862+00:00 ERROR 'ansible_navigator.image_manager.puller._log_message' Execution environment pull failed
2022-07-15T16:00:35.802375+00:00 ERROR 'ansible_navigator.image_manager.puller._log_message' Error response from daemon: Get "http://localhost/v2/": dial tcp [::1]:80: connect: connection refused
2022-07-15T16:00:42.066391+00:00 ERROR 'ansible_navigator.image_manager.puller._log_message' Execution environment pull failed
2022-07-15T16:00:42.066845+00:00 ERROR 'ansible_navigator.image_manager.puller._log_message' Error response from daemon: Get "http://localhost/v2/": dial tcp [::1]:80: connect: connection refused
